docs/.vuepress/config.js

```diff
@@ -25,7 +25,8 @@ const sidebars = {
       children: [
         ['/guide/relationships/getting-started', 'Getting Started'],
         ['/guide/relationships/one-to-one', 'One To One'],
-        ['/guide/relationships/one-to-many', 'One To Many']
+        ['/guide/relationships/one-to-many', 'One To Many'],
+        ['/guide/relationships/one-to-one-poly', 'One To One (Polymorphic)']
```

Comment:
This is tough one. So it's either we should place this docs within `one-to-one` page, or create a `Polymorphic` page.

I'm kinda more toward to create `Polymorphic` page, and list all polymorphic relations under there, since this is quite unique relationship.

@cuebit Any idea?

Reply:
Thanks for the feedback! I will move the content into a separate `Polymorphic` page as per discussion